packages/firestore/src/local/indexeddb_persistence.ts

```diff
@@ -352,9 +353,6 @@ export class IndexedDbPersistence implements Persistence {
     if (!this.started) {
       return Promise.resolve();
     }
-    // TODO(multitab): Similar to the zombied client ID, we should write an
-    // entry to Local Storage first to indicate that we are no longer alive.
-    // This will help us when the shutdown handler doesn't run to completion.
     this.started = false;
```

Comment:
Should you have added a markClientZombied() call?

Reply:
It's called in the window unload handler before all the rest of the stuff that runs on the AsyncQueue.

Reply:
Is that the only place shutdown() is called?

Reply:
If we decide we don't want/need to mark zombie for other shutdown() cases, that's fine...  but then the removeClientZombiedEntry() should only be in the unload handler as well.

Reply:
👍